### ISSUE TITLE
Explain that either foo/mod.rs or foo.rs is necessary 

### ISSUE THREAD

**Issue by Theemuts** [rust-lang/edition-guide#129]
The edition guide contains a chapter on Path Clarity (3.1.2), which explains that for a module foo with submodules, foo/mod.rs is no longer required. From the text, it is not clear foo.rs should be created instead.

**Comment by wa7sa34cx** [comment]
I agree with it. No more mod.rs! Because if you have a bunch of files open in your editor, you can clearly see their names, instead of having a bunch of tabs named mod.rs.
